timemachine/potentials/nonbonded.py

```diff
@@ -562,6 +562,93 @@ def coulomb_interaction_group_energy(q_ligand: Array, q_prefactors: Array) -> fl
 
 
 # utilities for efficiently recomputing energy as a function of ligand LJ parameters
+
+# note: two cases:
+# (1) only varying LJ eps parameters (can summarize using 1 float per ligand atom)
+# (2) simultaneously varying LJ eps and sig parameters (summarized using 20 floats per ligand atom)
+
+
+# case (1) : only varying ligand LJ eps parameters
+def lj_eps_prefactor_on_atom(x_i, x_others, sig_i, sig_others, eps_others, box=None, cutoff=1.2) -> float:
```

Comment:
In some of the prefactor code this defaults to cutoff=np.inf vs cutoff=1.2 . Is there a reason for this?

Reply:
oops, mistakenly used 1.2 in this PR, reverted to np.inf in https://github.com/proteneer/timemachine/pull/1212/commits/d5a57e56f89bfa501dc826ea37aebeee25a8e5e1